modules/dr-restoring-cluster-state.adoc

```diff
@@ -97,10 +97,51 @@ The output of this command should be empty. If it is not empty, wait a few minut
 $ sudo mv /var/lib/etcd/ /tmp
 ----
 
+.. If the `/etc/kubernetes/manifests/keepalived.yaml` file exists, follow these steps:
```

Comment:
I think marking these as optional would separate such steps better from the core steps.

> Optional: If the `/etc/kubernetes/manifests/keepalived.yaml` file exists on your local system, complete the following steps:

Reply:
I'm not sure that I agree. These steps are not optional if Keepalived is in use, so marking them as optional would convey the wrong idea, don't you think?

Reply:
Reading the "if" statement to me read like the step was optional. I do not see an option  for if "/etc/kubernetes/manifests/keepalived.yaml file" does not exist.

Reply:
Correct, the substeps only apply if the keepalived configuration exists. There are no extra steps required otherwise, as that means keepalived is not in use.

A lot of the information in this procedure depends on how the cluster is configured. :slightly_smiling_face: